apps/web/src/app/globals.css

```
@@ -96,111 +149,106 @@ html {
 }
 
 body {
   color: var(--fg);
   background: var(--bg-0);
-  font-family: var(--font-sans);
+  font-family: var(--font-ui);
+  font-size: var(--text-body);
+  line-height: 1.6;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.1)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)